src/Resources/views/Calls/macros.html.twig

```diff
@@ -88,3 +88,9 @@
         </div>
     {% endif %}
 {% endmacro %}
+
+{% macro render_curl(curl) %}
+    <h4>cURL</h4>
```

Comment:
You should add an `if` statement, otherwise this won't work with older profiles.